File: store-admin/src/main/java/org/delivery/storeadmin/domain/userorder/controller/model/UserOrderDetailResponse.java
package org.delivery.storeadmin.domain.userorder.controller.model;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.delivery.storeadmin.domain.storemenu.controller.model.StoreMenuResponse;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class UserOrderDetailResponse {

    private UserOrderResponse userOrderResponse;
    private List<StoreMenuResponse> storeMenuResponseList;

}

File: api/src/main/kotlin/org/delivery/api/domain/userorder/controller/model/UserOrderDetailResponse.kt
package org.delivery.api.domain.userorder.controller.model

import org.delivery.api.domain.store.controller.model.StoreResponse
import org.delivery.api.domain.storemenu.controller.model.StoreMenuResponse


data class UserOrderDetailResponse (
    var userOrderResponse: UserOrderResponse? = null, // 주문건
    var storeResponse: StoreResponse? = null, // 가게 정보
    var storeMenuResponses: List<StoreMenuResponse>? = null, // 메뉴 n개 정보
)
